src/pages/settings/SettingsButtons.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from "antd";
+import { t } from "i18next";
+import { FunctionComponent, useState } from "react";
+import SettingsDataHandler from "../../data/SettingsDataHandler";
+
+interface SettingsButtonProps {}
+
+export const SettingsButton: FunctionComponent<SettingsButtonProps> = () => {
+    const [reloadCount, setReloadCount] = useState(0);
+
+    const listener = () => {
+        setReloadCount(reloadCount + 1);
+    };
+    SettingsDataHandler.getInstance().addListener(listener);
+    const isDisabled = !SettingsDataHandler.getInstance().hasUnchangedChanges();
+    return (
+        <div
+            style={{
+                display: "flex",
+                gap: 8,
+                justifyContent: "flex-end",
+            }}
+        >
+            <Button disabled={isDisabled} onClick={() => SettingsDataHandler.getInstance().resetAll()}>
```

Comment:
disabling cancel might be a problem in the toniebox settings page. maybe there should be a third button "cancel".